controlsfx/src/main/java/org/controlsfx/control/table/FilterPanel.java

```diff
@@ -62,22 +59,12 @@
     private final FilteredList<FilterValue> filterList;
     private final TextField searchBox = new TextField();
     private boolean searchMode = false;

```

Comment:
This is only used in `initializeListeners`, maybe remove it?

Reply:
Removed ```searchMode``` entirely as it wasn't being used even in ```initializedListeners```.